traffic_ops/app/db/trafficvault/scripts/fill_expiration_and_provider.conf

```diff
@@ -0,0 +1,8 @@
+{
+	"dbname": "",
+	"hostname": "",
+	"user": "",
+	"password": "",
+	"port": 5432,
+	"ssl": false
+}
```

Comment:
Can you name this file with `.json`? Some editors aren't smart enough to figure out the grammar from the contents and rely on "file extensions".